ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml

```diff
@@ -37,6 +37,23 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: rosa-aws-sts
   timeout: 6h0m0s
+- as: data-path-9nodes
+  cron: 0 13 * * 6
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
+      CONFIG: config/standard-classic-rosa.yml
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.14"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "9"
+    test:
```

Comment:
`- ref: openshift-qe-perfscale-aws-data-path-sg`

Need to add this to ensure we can run network tests using hostNetwork.